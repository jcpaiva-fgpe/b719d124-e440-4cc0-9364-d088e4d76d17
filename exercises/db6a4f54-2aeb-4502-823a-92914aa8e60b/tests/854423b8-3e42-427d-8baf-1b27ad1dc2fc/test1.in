less border use factor
ly chiripiti
ing book cheer
a
