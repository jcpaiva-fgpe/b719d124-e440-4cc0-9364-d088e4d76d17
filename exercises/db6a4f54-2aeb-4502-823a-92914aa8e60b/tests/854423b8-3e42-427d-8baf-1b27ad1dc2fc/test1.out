borderless
useless
factorless
chiripitily
booking
cheering
